bayesian/utility/v2/sa_models.py

```diff
@@ -31,6 +31,13 @@ class Ecosystem(str, Enum):
     npm = 'npm'
 
 
+class RegistrationStatus(str, Enum):
```

Comment:
Can you use the ENUM in user_utils

Reply:
There is difference between status value used in Backbone flow and user utils, we need to have mapping somewhere. We can discuss on this in standup and close it.